cal/cal/helpers.py

```diff
@@ -113,9 +119,34 @@ def get_time_series(model, timezone='UTC', time_step='week', calendar_ids=None,
             else:
                 total += (events[i].end - events[i].start).total_seconds() / 3600
             i += 1
+
+        # For moving average. Basically iterates back and adds average
+        data_point += 1
+        period_total += total
+        if data_point % period == 0:
+            average = float(period_total) / period
```

Comment:
can u do from __future__ import division so that you don't have to use float here and various other places in this file?